modules/template/template.go

```diff
@@ -104,6 +105,15 @@ func NewFuncMap() []template.FuncMap {
 		"ThemeColorMetaTag": func() string {
 			return setting.ThemeColorMetaTag
 		},
+		"TabSizeClass": func(ec *editorconfig.Editorconfig, filename string) string {
+			if ec != nil {
+				def := ec.GetDefinitionForFilename(filename)
+				if def.TabWidth > 0 {
+					return fmt.Sprintf("tab-size-%d", def.TabWidth)
+				}
+			}
+			return "tab-size-8"
```

Comment:
@rugk This page has some statistics: https://ukupat.github.io/tabs-or-spaces/

I thought on 8 because it is backward compatible: its how it would be shown on Gogs today.

I will let @Unknwon decide this.